game/templates/game/replay.html

```
@@ -0,0 +1,100 @@
+            </div>
+        </div>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/chess.js/0.10.3/chess.min.js"></script>
+    <script src="https://unpkg.com/@chrisoakman/chessboardjs@1.0.0/dist/chessboard-1.0.0.min.js"></script>
+    <script src="{% static 'game/js/replay.js' %}"></script>
+</body>
+
+</html>
+</body>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </body> ]

(tag-pair)